rcqt.so/src/action.cpp

```diff
@@ -55,6 +55,7 @@
 #define JSON_CREATE_NODE_ERROR "JSON cannot create node"
 #define JSON_PKGCHK_NODE_NAME "pkgchk"
 #define PACKAGE "package"
+#define PACKAGELIST "packagelist"
```

Comment:
just another general comment , or a guideline for us to follow, to use static const variables instead of preprocessor directives , atleast going forward.

Reply:
Currently, we have preprocessor directives in RVS for property variables. Need to see if static const variables really does any value add.